### ISSUE TITLE
question about installment

### ISSUE THREAD

**Issue by spacewalk01** [gaugecam-dev/GRIME2#9]
There is a meter-stick beside detection board as shown in the image. I wonder if this meter-stick is used for calibration and line detection. Is it possible to install only board(ROI) without meter-stick?
![image](https://user-images.githubusercontent.com/6389915/139194218-4a549fd2-472d-40ae-a18e-b5cdd3295d7e.png)

**Comment by gaugecam** [comment]
No. That is completely unnecessary. It was placed in this scene for the convenience of the researchers but the calibration only requires the bow tie target, not the staff gauge (meter-stick).
